gravitee-am-repository/gravitee-am-repository-jdbc/src/main/resources/liquibase/changelogs/v3_19_0/3.19.0-schema-dictionaries.yml

```
@@ -0,0 +1,45 @@
+databaseChangeLog:
+  - changeSet:
+      id: 3.19.0-schema-dictionaries
+      author: GraviteeSource Team
+      changes:
+        - createTable:
+            tableName: dictionaries
```

As we name the object `I18nDictionary` at REST API level because we may have in the future other type of dictionaries, may be we could also name the table `i18n_dictionaries` for consistency
```suggestion
            tableName: i18n_dictionaries
```